Gridy/View/WhiteLayerView.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  GridOverlay.swift
+//  Gridy
+//
+//  Created by Alexandra Ivanova on 29/10/2020.
+//  Copyright © 2020 Alex Ivanova. All rights reserved.
+//
+
+import UIKit
+
+class WhiteLayerView: UIView {
+    var gridFrame: CGRect = .zero {
+        didSet { setup() }
+    }
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        self.clipsToBounds = true
+        self.alpha = 0.8
+        self.backgroundColor = UIColor(named: Constant.Color.secondaryLight)
+    }
+    
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
```

Comment:
FYI, this `init(frame: CGRect)` is the one called by Storyboard, you can copy past the setup from `init(frame: CGRect)` here or leave it empty and do the setup in Storyboard.